documentation/src/main/wiki/PerformanceTestsMethodology.md

```diff
@@ -0,0 +1,99 @@
+# Performance Tests Methodology
+
+## Introduction
+This page describes the methodology of Knot.x performance tests.
+
+## Goals
+The goals of the tests conducted are as follows:
+* Assess system performance.
+* Assess system stability.
+
+### Metrics
+During our tests, we focus on the following metrics which, when correlated, allow us to come up with number of observations about the system:
+* Throughput.
+* Response time - 90% Line.
+* Quantity of Virtual Users.
+* Number of errors (HTTP response codes > 400, Knot.x logs).
+* Infrastructure resources (cpu, memory, disk, network).
+* Application resources (jmx heap size, garbage collections).
+
+### KPIs
+* 1 second for response time (90% Line).
+* Error occurrences.
+* Stability:
+    * No memory leaks.
+    * No errors (logs, responses).
+    * Infrastructure resources OK.
+    * Application resources OK.
+	
+## Test Environment
+Test Environment is set up on AWS Cloud.
+![Test environment](assets/knotx-test-environment.png)
+
+### System under test
+* 1 x Apache - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
+* 1 x Knot.x - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
+* 1 x Mocks - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
+* 1 x AEM - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth.centos 6.8.
```

Comment:
There is no such machine on the test environment structure. Also AEM is not mentioned in tests. Looks like some copy paste?